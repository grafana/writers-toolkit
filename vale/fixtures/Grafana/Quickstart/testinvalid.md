Don't use:

- quick start
