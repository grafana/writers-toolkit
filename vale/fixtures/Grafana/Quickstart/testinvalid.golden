testinvalid.md:3:3:Grafana.Quickstart:Use the compound adjective without a hyphen whether the noun is implied or explicit. For example, you can use _quickstart guide_ or just _quickstart_.
